core/async_job/queue.ex

```diff
@@ -125,28 +116,70 @@ defmodule AntikytheraCore.AsyncJob.Queue do
     end
   end
 
+  defp move_job_running_too_long(q, target_job_info, _now_millis, _threshold_time) when is_nil(target_job_info), do: q
```

Comment:
Following is more simple and general.
```
defp move_job_running_too_long(q, nil, _now_millis, _threshold_time), do: q
```

Reply:
I fixed it in https://github.com/access-company/antikythera/commit/43cce81c61461e12e362aa199679425fc58dcaa3